onnx/helper.py

```diff
@@ -73,6 +73,14 @@
 
 VersionMapType = Dict[Tuple[str, int], int]
 
+# TODO (https://github.com/microsoft/onnxruntime/issues/14932): Get max supported version from onnxruntime directly
+ORT_MAX_ONNX_OPSET_SUPPORTED_VERSION = 18
```

Comment:
I would probably make it as environment variable in the CI job rather than mentioning ORT in the repository.

Reply:
Although we will need to manually bump the supported versions as environment variables in 4 CIs, I cannot think of other better thought. I have added them as environment variables.